src/ai/backend/manager/idle.py

```diff
@@ -422,6 +422,11 @@ async def check_session(self, session: Row, dbconn: SAConnection) -> bool:
             if occupied_slots[slot] == 0:
                 unavailable_resources.update(self.slot_resource_map[slot])
 
+        if ("cuda.device" in occupied_slots.keys()) or ("cuda.shares" in occupied_slots.keys()):
+            if (occupied_slots["cuda.device"] == 1) or (occupied_slots["cuda.shares"] == 1):
```

Comment:
This line will raise an error if `occupied_slots` have only one of `cuda.device` or `cuda.shares`.

Reply:
If `occupied_slots["cuda.shares"]` is not zero and `1`, what will happen?

Reply:
We also need to be as much as general here since there can be many other types of CUDA slots, for example, "cuda.mig.*", which may be present in MIG-enabled environment.

Reply:
I made a commit to resolve the above-mentioned issues. Please check if this works.
https://github.com/lablup/backend.ai-manager/pull/432/commits/14266d77231e1dde148511f19b50219cb3e25122